lxd/storage/backend_lxd.go

```diff
@@ -2451,6 +2453,22 @@ func (b *lxdBackend) updateVolumeDescriptionOnly(project string, volName string,
 		}
 	}
 
+	// Get content type.
+	dbContentType, err := VolumeContentTypeNameToContentType(curVol.ContentType)
+	if err != nil {
+		return err
+	}
+
+	contentType, err := VolumeDBContentTypeToContentType(dbContentType)
+	if err != nil {
+		return err
+	}
+
+	// Validate config.
+	vol := b.newVolume(drivers.VolumeType(curVol.Type), contentType, volName, newConfig)
+
+	b.state.Events.SendLifecycle(project, lifecycle.StorageVolumeUpdated.Event(vol, string(vol.Type()), project, op, log.Ctx{"type": curVol.Type}))
```

Comment:
I don't think we actually care about the `type` property on the volumes. You can't have two volumes of different types with the same name, so it can be treated as if it was configuration and not be logged every time.

Maybe the one exception where we should include it is at creation time as that may be useful to know then.

Reply:
Alright, I swapped it out for nils on all the `Updated` events and added it to all the `Created` events.